apps/admin_api/priv/swagger/wallet/request_bodies.yaml

```diff
@@ -44,6 +44,23 @@ WalletUserBody:
         example:
           id: usr_01cc02x0v98qcctvycfx4vsk8x
 
+BalanceAllForWalletBody:
+  description: The parameters to use for listing the balances for a wallet
```

Comment:
Can we specify that the pagination params are based on `token` to make it clear?